mm2src/lp_ordermatch.rs

```diff
@@ -4263,6 +4264,41 @@ async fn get_max_volume(ctx: &MmArc, my_coin: &MmCoinEnum, other_coin: &MmCoinEn
     ))
 }
 
+pub async fn check_balance_update_loop(ctx: MmWeak, ticker: String) {
+    let mut current_balance = None;
+    loop {
+        Timer::sleep(10.).await;
+        let ctx = match MmArc::from_weak(&ctx) {
+            Some(ctx) => ctx,
+            None => return,
+        };
+
+        match lp_coinfind(&ctx, &ticker).await {
+            Ok(Some(coin)) => {
+                let storage = MyOrdersStorage::new(ctx.clone());
```

Comment:
Maker orders are primarily stored in memory. Please use `OrdermatchContext::maker_orders` instead of `MyOrdersStorage`.